downloads/platform.md

```diff
@@ -76,9 +76,16 @@ You can uninstall Julia by deleting Julia.app and the packages directory in `~/.
 
 ## Linux and FreeBSD
 
-It is strongly recommended that the official generic binaries from the downloads page be used to install Julia on Linux and FreeBSD.
+It is strongly recommended that the official generic binaries from the downloads page be used to install Julia on Linux and FreeBSD. The following
+command downloads the latest Julia version and makes a symlink to allow the `julia` command to find the binary:
```

Comment:
The given command doesn't actually download the latest Julia version (it happens to download the most recent 1.4 release currently, but will soon be out of date). Might be worth emphasizing that it's an example for a particular version.